src/main/license/source-inclusions.yaml

```diff
@@ -20,10 +20,9 @@
 # extras file for org.heneveld.license-audit-maven-plugin
 # listing projects from which *source* files are included
 
-- id: jquery-core:1.7.2
-- id: swagger-ui:2.1.4
-# we use other versions of the above in other projs
-
```

Comment:
i think this is an accidental reversion.  the version numbers should be included.  (you might want also to auto-regen the license file -- that also has some wanted new stuff removed but it's not a big deal.)